src/bin/ion/commands/cat.rs

```diff
@@ -27,6 +28,7 @@ impl IonCliCommand for CatCommand {
     fn configure_args(&self, command: Command) -> Command {
         command
             .alias("dump")
+            .arg(arg!(-t --"detect-timestamps" "Preserve Ion timestamps when going from Ion to JSON to Ion"))
```

Comment:
I'm thinking that this flag doesn't belong with `cat`, only with `from json`. That's up to taste, I suppose, but I'd rather not have `cat` change the semantics of any data it handles.